diff --git a/node_modules/react-native-blurhash/android/src/main/java/com/mrousavy/blurhash/BlurhashViewManager.kt b/node_modules/react-native-blurhash/android/src/main/java/com/mrousavy/blurhash/BlurhashViewManager.kt
index 8a25c1a..fe38982 100644
--- a/node_modules/react-native-blurhash/android/src/main/java/com/mrousavy/blurhash/BlurhashViewManager.kt
+++ b/node_modules/react-native-blurhash/android/src/main/java/com/mrousavy/blurhash/BlurhashViewManager.kt
@@ -23,7 +23,7 @@ internal class LoadStartEvent(
   override fun getEventData(): WritableMap = Arguments.createMap()
 
   companion object {
-    const val EVENT_NAME = "topBlurhashLoadStart"
+    const val EVENT_NAME = "topLoadStart"
   }
 }
 
@@ -36,7 +36,7 @@ internal class LoadEndEvent(
   override fun getEventData(): WritableMap = Arguments.createMap()
 
   companion object {
-    const val EVENT_NAME = "topBlurhashLoadEnd"
+    const val EVENT_NAME = "topLoadStart"
   }
 }
 
@@ -51,7 +51,7 @@ internal class LoadErrorEvent(
       Arguments.createMap().apply { putString("message", message) }
 
   companion object {
-    const val EVENT_NAME = "topBlurhashLoadError"
+    const val EVENT_NAME = "topLoadStart"
   }
 }
 
